packages/playground/_stories/main/SplitButton/SplitButton.stories.ts

```diff
@@ -40,6 +40,28 @@ Basic.args = {
 	default: "Default",
 };
 
+export const SplitButtonWithMenu = Template.bind(this);
+SplitButtonWithMenu.args = {
+	default: "Open Menu",
+};
+SplitButtonWithMenu.decorators = [
+	(story) => {
+		return html`
+		<ui5-menu id="menuInSplitBtnDefaultAction">
+		<ui5-menu-item text="Edit" icon="add"></ui5-menu-item>
+		<ui5-menu-item text="Save" icon="save"></ui5-menu-item>
+		<ui5-menu-item text="Delete" icon="delete"></ui5-menu-item>
+		</ui5-menu>
+	${story()}
+	<script>
+		const splitBtnWithMenuDefaultAction = document.querySelector("ui5-split-button");
```

Comment:
If you go to another story and then comeback to the newly added one an error will appear in the console because of double declaration of splitBtnWithMenuDefaultAction. @gmkv found a workaround with that by changing const to var so let's use it here.